lib/solargraph/shell.rb

```diff
@@ -173,6 +173,7 @@ def typecheck *files
       }
       puts "Typecheck finished in #{time.real} seconds."
       puts "#{probcount} problem#{probcount != 1 ? 's' : ''} found#{files.length != 1 ? " in #{filecount} of #{files.length} files" : ''}."
+      # "
```

Comment:
I'm working on support for `textDocument/semanticTokens`. Hopefully it'll remove the need for this workaround.